python/troubleshooting_guide.md

```diff
@@ -32,4 +32,16 @@ Blocking the thread can significantly delay execution, potentially leading to a
 ##### Possible solutions
 - configure the workers to run activities in a different thread  https://github.com/temporalio/sdk-python?tab=readme-ov-file#synchronous-multithreaded-activities
 - use `asyncio.to_thread(my_blocking_func)` or `asyncio.get_running_loop().run_in_executor(None, my_blocking_func)` to run the blocking code inside the async function.
-- if you are unsure if an activity makes blocking calls, convert the async activities into synchronous ones ([docs](https://docs.temporal.io/develop/python/python-sdk-sync-vs-async#when-should-you-use-async-activities)).
\ No newline at end of file
+- if you are unsure if an activity makes blocking calls, convert the async activities into synchronous ones ([docs](https://docs.temporal.io/develop/python/python-sdk-sync-vs-async#when-should-you-use-async-activities)).
+- see the ["Registering Activities" page of the Python 101 course](https://temporal.talentlms.com/catalog/info/id:143) for guidance on sync and async activities.
+- see the [API docs for the worker initialization](https://python.temporal.io/temporalio.worker.Worker.html#__init__) to understand the configuration options.
+
+## Memory Leak
+
+If you are seeing a memory leak, then
+**make sure you are passing through your modules into the workflow sandbox**.
+
+1. Separate your activity code and your workflow code into different files.
```

Comment:
@GSmithApps thanks 

is there a link to doc/issue/slack discussion... we can add here? Same for the next point